e2e/shortcut-action/shortcut-action.e2e-spec.ts

```diff
@@ -58,19 +58,29 @@ describe('Shortcut-action App', function() {
   });
 
   it(`should move row up using 'mod+shift+up' shortcut`, () => {
-    let inputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.1.experiment'), 'textarea');
-    let inputElemValue = inputElem.getAttribute('value');
-    inputElem.sendKeys(protractor.Key.chord(mod, protractor.Key.SHIFT, protractor.Key.UP));
-    let movedUpInputElem = page.getChildOfElementByCss(page.getElementById('accelerator_experiments.0.experiment'), 'textarea');
-    expect(movedUpInputElem.getAttribute('value')).toEqual(inputElemValue);
+    let currentFirstRow = page.getElementArrayById('accelerator_experiments.0');
```

Comment:
The array with the elements of the first row. The `toEqual()` is waiting for the promises to be resolves an then it compares their values.